### PR TITLE
Add Ember 2.6.2 example

### PR DESCRIPTION
**Pull request by stephencelis**
Have been working with Ember at work a bit and was curious how it compared. Performance between runs varied a bit in my experience, but the general ranks were the same.

screen shot 2016-07-17 at 3 42 45 pm

The above is from a Chrome run. Safari (Technology Preview) has different results, where Angular is much slower and unoptimized React and Ember are faster.

I'm not sure what's expected to produce another, more "optimized" version.

### CODE REVIEW

**Comment by evancz** [comment]
Thank you! :D

On optimization, I guess stuff like [this](http://codesandnotes.com/ember-js/tips-for-optimizing-ember.js-performance/), though I don't have enough experience to know how to actually do those things.

I'm thinking it could be cool to write about "how hard is it to optimize the obvious solution?" so the story here may be that it is quite difficult.

Anyway, thanks, excited to look through the code!

File: benchmarks/todomvc/ember-2.6.2/app/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <body>
+    {{content-for "body"}}
+
+    <script src="assets/vendor.js"></script>
+    <script src="assets/todomvc.js"></script>
+
+    {{content-for "body-footer"}}
+
+    <script>
+      window.requestAnimationFrame = window.webkitRequestAnimationFrame = function(callback) {
```
Comment (evancz) [inline benchmarks/todomvc/ember-2.6.2/app/index.html]: Was this straight from the TodoMVC version? Do you know if they use this on each frame? As an aside, we tried to make `requestAnimationFrame` a dummy in other frameworks with:

``` javascript
window.requestAnimationFrame = function(callback) { callback(performance.now()); }
```

But I guess that may not be enough.

Comment (stephencelis) [inline benchmarks/todomvc/ember-2.6.2/app/index.html]: I blindly stole this from the Angular example. I'm not sure Ember uses `requestAnimationFrame` at all. Will do some digging and remove this if I can.

Comment (evancz) [inline benchmarks/todomvc/ember-2.6.2/app/index.html]: Ah, gotcha! I was confused. I think the only change is that it should be above all the other assets. That way if someone says `var rAF = requestAnimationFrame` they don't store the real one away.

Comment (stephencelis) [inline benchmarks/todomvc/ember-2.6.2/app/index.html]: Did a quick grep and Ember doesn't appear to use it at all. Will get rid of it!

**Comment by evancz** [comment]
Great, just finished looking through and trimming a few files out. Thanks for doing this!

I now know what you mean about not being sure how to optimize this. I have a sense of how things fit together, but have no idea where I have opportunities for optimization. If you have any coworkers with some experience / wisdom on this topic, I'd love to learn more about it.

**Pull request merged**